test/test-kubernetes.js

```diff
@@ -29,6 +29,7 @@ const scaffolderSampleSwift = scaffolderSample.getJson('SWIFT');
 const scaffolderSampleJava = scaffolderSample.getJson('JAVA');
 const scaffolderSampleSpring = scaffolderSample.getJson('SPRING');
 const scaffolderSamplePython = scaffolderSample.getJson('PYTHON');
+const scaffolderSampleGo = scaffolderSample.getJson('GO');
```

Comment:
the tests for Go seem light compared to other languages. Can you revisit this and see if there are other meaningful tests we can add?

Reply:
added more!